elastic/kibana
# [Cloud Security] Added investigate in timeline to graph investigation

## Summary

This PR includes the following changes to the graph investigation component:
- Add "Investigate in timeline" button to the graph in the expanded flyout (applying the same filters to the timeline)
- Move graph control buttons to the right and align their design


https://github.com/user-attachments/assets/6e7c59a7-150d-4046-8e3d-14790b8f6d4d

<details>
<summary>New storybook stories 📹 </summary>

https://github.com/user-attachments/assets/adf64f7f-3bd3-499e-b9ba-f8040df729e7

</details>

**How to test:**

To test this PR using storybook (alternatively access to storybooks attached to this build)

```
yarn storybook cloud_security_posture_packages
```

To test e2e:

- Enable the feature flag 

`kibana.dev.yml`:

```yaml
uiSettings.overrides.securitySolution:enableVisualizationsInFlyout: true
xpack.securitySolution.enableExperimental: ['graphVisualizationInFlyoutEnabled']
```

- Load mocked data:

```bash
node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/logs_gcp_audit \ 
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601

node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/security_alerts \
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601
```

- Make sure you include data from Oct 13 2024. (in the video I use Last year)

To run FTR tests:

```
yarn test:ftr:server --config x-pack/test/cloud_security_posture_functional/config.ts
yarn test:ftr:runner --config x-pack/test/cloud_security_posture_functional/config.ts --grep="Graph visualization"
```

### Checklist
